.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -214,25 +216,75 @@ private object GutenbergUploadSourceMapsToSentry: BuildType() {
 
 		steps {
 			bashNodeScript {
-				name = "Upload source maps to Sentry"
-				scriptContent = """
-					# Install the Sentry CLI binary
-					curl -sL https://sentry.io/get-cli/ | bash
+				name = "Install Sentry CLI"
+				scriptContent = "curl -sL https://sentry.io/get-cli/ | bash"
+			}
 
+			bashNodeScript {
+				name = "Upload Gutenberg source maps to Sentry"
```

Comment:
Should we rename this to something more general now that it also uploads maps to other libraries? Maybe `Upload source amps to Sentry` to keep things simple (since it's already part of the Calypso Plugins project in TeamCity).

Reply:
I think Gutenberg makes sense here because we have an individual build step for each different plugin. (e.g. `sentry-cli` is called a total of 4 times now)